modules/ROOT/pages/release-notes/release-notes59.adoc

```diff
@@ -362,28 +372,31 @@ For information on:
 * The `formatter.formatChanged` API would sometimes not run the callback the first time the format was removed.
 * Base64 encoded images with spaces or line breaks in the data URI were not displayed correctly. Patch contributed by RoboBurned
 
+[[securityfixes]]
 == Security fixes
 
 {productname} 5.9 provides fixes for the following security issues:
 
 * Inserting certain HTML content into the editor could result in invalid HTML once parsed. This caused a medium severity Cross Site Scripting (XSS) vulnerability. Tiny Technologies would like to thank William Bowling for discovering this vulnerability.
 * The `.war` files for the premium self-hosted server-side components have been updated to address various high and medium severity security issues.
 
+[[deprecatedfeatures]]
 == Deprecated features
 
 The following features have been deprecated with the release of {productname} 5.9:
 
-* <<thespellchecker_select_languagesoption,The `spellchecker_select_languages` option>>.
-* <<thebbcodebbcodeplugin,The BBCode (`bbcode`) plugin>>
-* <<thefullpagefullpageplugin,The Full Page (`fullpage`) plugin>>
-* <<thelegacyoutputlegacyoutputplugin,The Legacy Output (`legacyoutput`) plugin>>
-* <<reminderthefreetinymcespellcheckerplugin,Reminder: The free TinyMCE Spell Checker plugin>>
+* xref:thespellchecker_select_languagesoption[The `spellchecker_select_languages` option]
+* xref:thebbcodebbcodeplugin[The BBCode (`bbcode`) plugin]
+* xref:thefullpagefullpageplugin[The Full Page (`fullpage`) plugin]
+* xref:thelegacyoutputlegacyoutputplugin[The Legacy Output (`legacyoutput`) plugin]
```

Comment:
This xref doesn't resolve as the heading below is missing the `thelegacyoutputlegacyoutputplugin` id. e.g:
```
[[thelegacyoutputlegacyoutputplugin]]
=== The Legacy Output (`legacyoutput`) plugin
```

Reply:
done